Documentation/Usage.md

```diff
@@ -4,40 +4,40 @@
 
 * [Overview](README.md)
 * [**Usage**](Usage.md)
-  * [Create a Package](#create-a-package)
-    * [Create a library package](#create-a-library-package)
-    * [Create an executable package](#create-an-executable-package)
-  * [Define Dependencies](#define-dependencies)
-  * [Publish a package](#publish-a-package)
-  * [Require System Libraries](#require-system-libraries)
-  * [Packaging legacy code](#packaging-legacy-code)
-  * [Handling version-specific logic](#handling-version-specific-logic)
+  * [Creating a Package](#creating-a-package)
+    * [Creating a Library Package](#creating-a-library-package)
+    * [Creating an Executable Package](#creating-an-executable-package)
+  * [Defining Dependencies](#defining-dependencies)
+  * [Publishing a Package](#publishing-a-package)
+  * [Requiring System Libraries](#requiring-system-libraries)
+  * [Packaging Legacy Code](#packaging-legacy-code)
+  * [Handling Version-specific Logic](#handling-version-specific-logic)
   * [Editable Packages](#editable-packages)
```

Comment:
As you're editing the different section headings anyway, can we make all of them start with a gerund, so they are all actionable phrases, such as `Creating a Package` to be more consistent?
For example, `Resolving Versions` instead of `Resolved Versions`

Reply:
I believe these have been updated in the Usage.md file, and for a few other headings.